README.md

```diff
@@ -1,6 +1,7 @@
 # CodeIgniter 4 Development
 
-[![Build Status](https://github.com/codeigniter4/CodeIgniter4/workflows/PHPUnit/badge.svg)](https://github.com/codeigniter4/CodeIgniter4/actions?query=workflow%3A%22PHPUnit%22)
+[![PHPUnit](https://github.com/codeigniter4/CodeIgniter4/workflows/PHPUnit/badge.svg)](https://github.com/codeigniter4/CodeIgniter4/actions/workflows/test-phpunit.yml)
+[![Psalm](https://github.com/codeigniter4/CodeIgniter4/actions/workflows/test-psalm.yml/badge.svg)](https://github.com/codeigniter4/CodeIgniter4/actions/workflows/test-psalm.yml)
```

Comment:
If we are going to add the badge for psalm, should we also do the same for the others (like phpstan, cs-fixer, rector)?

Reply:
I'm all for showing off our badge swag! But maybe we should move towards a more organized format like a table or something?